README.md

```diff
@@ -135,6 +134,17 @@ duplicate([1,2,3,4,5]); // [1,2,3,4,5,1,2,3,4,5]
 * Why would you use something like the `load` event? Does this event have disadvantages? Do you know any alternatives, and why would you use those?
 * Explain what a single page app is and how to make one SEO-friendly.
 * What is the extent of your experience with Promises and/or their polyfills?
+* What are some of the advantages/disadvantages of writing JavaScript code in a language that compiles to JavaScript?
+
+#### Testing and Performance Questions:
```

Comment:
Hmm... I would probably lump them together under the umbrella of "good code" or something like that but, yeah, you're right. They can be split into 2 sections.